lib/Dialect/FIRRTL/Import/FIRAnnotations.cpp

```diff
@@ -91,6 +90,45 @@ splitTarget(StringRef target, MLIRContext *context) {
   return {targetBase, ArrayAttr::get(context, annotationVec)};
 }
 
+/// Split out non-local paths
+SmallVector<std::tuple<std::string, std::string, std::string>>
+expandNonLocal(StringRef target) {
+  SmallVector<std::tuple<std::string, std::string, std::string>> retval;
+  StringRef prefix;
+  std::tie(prefix, target) = target.split('|');
```

Comment:
Nit: this is really the "circuit" or "circuitTarget". Consider:

```suggestion
  StringRef circuit;
  std::tie(circuit, target) = target.split('|');
```